VirtualList/tests/useEvent-native-specs.js

```diff
@@ -0,0 +1,261 @@
+import '@testing-library/jest-dom';
+import {act, fireEvent, render, screen} from '@testing-library/react';
+
+import Item from '../../Item';
+import VirtualList from '../VirtualList';
+
+const focus = (elm) => fireEvent.focus(elm);
+
+const keyDownUp = (keyCode) => (elm) => {
+	fireEvent.keyDown(elm, {keyCode});
+	fireEvent.keyUp(elm, {keyCode});
+};
+
+const pressLeftKey = keyDownUp(37);
+const pressRightKey = keyDownUp(39);
+const pressUpKey = keyDownUp(38);
+const pressDownKey = keyDownUp(40);
+const pressPageDownKey = keyDownUp(34);
+
+describe('VirtualList useEvent with native `scrollMode`', () => {
+	let
+		clientSize,
+		currentFocusIndex,
+		dataSize,
+		handlerOnFocus,
+		items,
+		itemSize,
+		renderItem;
+
+	beforeEach(() => {
+		clientSize = {clientWidth: 1280, clientHeight: 720};
+		currentFocusIndex = -1;
+		dataSize = 200;
+		items = [];
+		itemSize = 60;
+
+		handlerOnFocus = (index) => () => {
+			currentFocusIndex = index;
+		};
+
+		renderItem = ({index, ...rest}) => { // eslint-disable-line enact/display-name
+			return (
+				<Item {...rest} onFocus={handlerOnFocus(index)}>
+					{items[index].name}
+				</Item>
+			);
+		};
+
+		for (let i = 0; i < dataSize; i++) {
+			items.push({name: 'Account ' + i});
+		}
+	});
+
+	afterEach(() => {
+		clientSize = null;
+		dataSize = null;
+		items = [];
+		itemSize = 60;
+		items = null;
```

Comment:
Is assigning null to items which was previously assigned an empty array intended?